modules/scripts/startup-script/files/startup-script-stdlib-head.sh

```diff
@@ -30,23 +30,24 @@ readonly E_RUN_OR_DIE=5
 readonly E_MISSING_MANDATORY_ARG=9
 readonly E_UNKNOWN_ARG=10
 
+SCRIPT_COMPLETE_FILE="/run/startup_script_msg"
```

Comment:
My instinct here would be `/run/ghpc/startup_script_msg`. Do you have a reason not to? Don't make change yet, I intend this to be start of discussion.

Reply:
It would also be nice if this could be defined once and sourced somehow. Please investigate.

Reply:
If possible, define this once. If not (or not easy/robust), then resolve this discussion.

Let's not consider any changes to location for now. We have an open task to better adopt [FHS](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html) best practices in Toolkit generally.

Reply:
I can't think of any easy ways to do this other than having the startup script modify the profile.d script which does not seem robust.  I will continue to think about it.